docs/guides/platform/Upgrade_IBM_i_Modernization_Engine_for_Lifecycle_Integration.md

```diff
@@ -1,29 +1,57 @@
 # Upgrade IBM i Modernization Engine for Lifecycle Integration
 
-This document discusses topics related to IBM i Developer Tool.
+## How to upgrade from Merlin v1 to v2
+To upgrade Merlin v1 to v2, the first step is to backup your existing data. Make sure that all of your work is backed up to a github (or some other source control system) repository before doing any of the next steps.
```

Comment:
what kind of data is this talking about?  all users' source they are developing or specific to the person doing the upgrade?
change 'github' to just 'git' unless there is specific reasons for a particular git repo provider

Reply:
Done